test/git/scenarios.json

```diff
@@ -43,5 +43,50 @@
                 "ppa": "false"
             }
         }
+    },
+    "install_git_from_src_centos-7": {
+        "image": "centos:centos7",
+        "features": {
+            "git": {
+                "version": "latest",
+                "ppa": "false"
+            }
+        }
+    },
+    "install_git_from_src_alma-8": {
+        "image": "almalinux:8",
+        "features": {
+            "git": {
+                "version": "latest",
+                "ppa": "false"
+            }
+        }
+    },
+    "install_git_from_src_alma-9": {
+        "image": "almalinux:9",
+        "features": {
+            "git": {
+                "version": "latest",
+                "ppa": "false"
+            }
+        }
+    },
+    "install_git_from_src_alma-8-minimal": {
+        "image": "almalinux:8-minimal",
+        "features": {
+            "git": {
+                "version": "latest",
+                "ppa": "false"
+            }
+        }
+    },
+    "install_git_from_src_alma-9-minimal": {
```

Comment:
Appreciate all these test scenarios, thank you! 

Can we also add similar scenarios for other distros this PR supports? (eg. fedora, mariner, rocky etc). Feel free to refer to https://github.com/devcontainers/features/blob/main/test/common-utils/scenarios.json

Reply:
Will do.

Reply:
Commit cba4c71 add requested tests and resolves this request.